src/iomei.cpp

```diff
@@ -2484,7 +2484,7 @@ void MEIOutput::WriteKeySig(pugi::xml_node currentNode, KeySig *keySig)
     // Only write att values if representing an attribute or in MEI basic
     if (!this->IsTreeObject(keySig)) {
         InstKeySigDefaultAnl attKeySigDefaultAnl;
-        // Broken in MEI 4.0.2 - waiting for a fix
```

Comment:
Why are you changing this? We are still waiting for a fix, aren't we?

Reply:
As far as I know there is now version 4.0.2?